test/PackageVersionsTest/E2EInstallerTest.php

```diff
@@ -289,6 +349,29 @@ private function writeComposerJsonFile(array $config, string $directory) : void
         );
     }
 
+    private function writeSymfonyPackageVersionUsingFile(string $directory) : void
+    {
+        file_put_contents(
+            $directory . '/use-package-versions.php',
+            <<<'PHP'
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+echo \PackageVersions\Versions::getVersion('symfony/process');
+PHP
+        );
+    }
+
+    private function assertSymfonyPackageVersionsIsUsable(string $directory) : void
+    {
+        exec(PHP_BINARY . ' ' . escapeshellarg($directory . '/use-package-versions.php'), $output, $exitCode);
+
+        self::assertSame(0, $exitCode);
+        self::assertCount(1, $output);
+        self::assertRegExp('/^v?5\\..*\\@[a-f0-9]*$/', $output[0]);
```

Comment:
Probably `self::assertStringStartsWith('v5.0.0', ...)`, since the rest of the regex is not relevant to this scenario

Reply:
Sounds valid, will fix that too